polkadot/node/core/runtime-api/src/tests.rs

```diff
@@ -286,6 +287,13 @@ impl RuntimeApiSubsystemClient for MockSubsystemClient {
 	async fn disabled_validators(&self, _: Hash) -> Result<Vec<ValidatorIndex>, ApiError> {
 		todo!("Not required for tests")
 	}
+
+	async fn claim_queue(
+		&self,
+		_: Hash,
+	) -> Result<BTreeMap<CoreIndex, VecDeque<ParasEntry<BlockNumber>>>, ApiError> {
```

Comment:
We should not expose `ParasEntry` here. That is an internal type that should not matter much outside of the scheduler. A `Vec<ParaId>` should be good enough.